hotshot-state-prover/src/service.rs

```diff
@@ -315,7 +368,37 @@ pub async fn submit_state_and_proof(
     // prepare the input the contract call and the tx itself
     let proof: ParsedPlonkProof = proof.into();
     let new_state: ParsedLightClientState = public_input.into();
-    let tx = contract.new_finalized_state(new_state.into(), proof.into());
+
+    let mut tx = contract.new_finalized_state(new_state.into(), proof.into());
+
+    // TODO: (alex) centralize this to a dedicated ethtxmanager with gas control
+    // frugal gas price: set to SafeGasPrice based on live price oracle
+    // safe to be included with low priority position in a block.
+    // ignore this if etherscan fail to return
+    if let Ok(res) =
```

Comment:
If this call returns `Err` it looks like we silently ignore it? We should log the error if this happens

Reply:
done in [db9ab26](https://github.com/EspressoSystems/espresso-sequencer/pull/2148/commits/db9ab26f48ec7b4f2f37faa0cc815d7f20c5ae6d)